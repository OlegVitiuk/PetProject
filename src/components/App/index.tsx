import React from "react";
import { BrowserRouter as Router, Route } from "react-router-dom";
import Home from "routes/Home";
import Navigation from "components/Navigation";
import Pay from "routes/Pay";
import SignIn from "routes/SignIn";
import SignUp from "routes/SignUp";
import styles from "./app.module.scss";

const App: React.SFC = () => {
  return (
    <div className={styles.appContainer}>
      <Router>
        <Navigation />
        <Route exact={true} path="/" component={SignIn} />
        <Route exact={true} path="/sign-up" component={SignUp} />
        <Route exact={true} path="/home" component={Home} />
        <Route exact={true} path="/pay" component={Pay} />
      </Router>
    </div>
  );
};

export default App;
