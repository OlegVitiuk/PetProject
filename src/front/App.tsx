import React from "react";
import "./App.css";

const App: React.FC = () => {
  return (
    <div className="App">
      <header className="App-header">header app</header>
    </div>
  );
};

export default App;

// interface AppProps {
//   chat: ChatState;
//   system: SystemState;
// }
