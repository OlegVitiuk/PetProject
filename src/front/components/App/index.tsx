import React from "react";
import CharacterList from "../Ð¡haracterList";
import "./App.css";

const App: React.SFC = () => {
  return (
    <div className="App">
      <header className="App-header">header app</header>
      <CharacterList />
    </div>
  );
};

export default App;
