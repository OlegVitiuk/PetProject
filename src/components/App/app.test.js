import React from "react";
import { shallow } from "enzyme";
import App from ".";

describe("App component", () => {
  it("render App component", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
