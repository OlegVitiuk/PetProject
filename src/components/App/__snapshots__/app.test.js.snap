// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component render App component 1`] = `ShallowWrapper {}`;

exports[`SignIn component render App component 1`] = `ShallowWrapper {}`;

exports[`render App component 1`] = `ShallowWrapper {}`;

exports[`renders without crashing 1`] = `ShallowWrapper {}`;
